README.md

```diff
@@ -1,37 +1,28 @@
 # Zapier Apify Integration
-<a href="https://travis-ci.com/apifytech/apify-zapier-integration?branch=master"><img src="https://travis-ci.com/apifytech/apify-zapier-integration.svg?branch=master" alt="Build Status" style="display:inherit;"></a>
 
 The Apify integration to Zapier allows you to connect Apify platform with 600+ app.
 
-## How it works
+## Resources
 
-You can follow [Apify integration](https://zapier.com/apps/Apify/integrations) page on Zapier platform.
+* [Apify documentation](https://docs.apify.com/tutorials/integrations#get-started)
+* [Getting started tutorial](https://help.apify.com/en/articles/3034235-getting-started-with-apify-integration-for-zapier)
+* [Apify integration](https://zapier.com/apps/Apify/integrations) page on Zapier platform
 
 
 ## Development
 
+If you are interested in adding a new feature or fix a bug in integration, feel free to open PR.
+
 ### Tests
 
-You need to specify your Apify API token before you run tests.
-You can do in root dir using file `.env`.
+You will need your Apify API token before you run tests.
+You can find the token [on your Apify account under the integration page](https://my.apify.com/account#/integrations).
```

Comment:
`on your Apify account under the integration page` => `on the Integrations page of your Apify account`